src/Generation/GapicClientV2Generator.php

```diff
@@ -172,6 +177,44 @@ private function serviceScopes(): PhpClassMember
             ->withValue(AST::array($this->serviceDetails->defaultScopes->toArray()));
     }
 
+    private function magicAsyncDocs(): PhpDoc
+    {
+        $methodDocs = $this->serviceDetails->methods
+            ->filter(fn($m) => !$m->isStreaming())
+            ->map(fn($m) => PhpDoc::method(
+                $m->methodName . "Async",
+                $this->ctx->type(Type::fromName(PromiseInterface::class))->type->name,
+                $m->requestType->name, // the request type will already be imported for the sync variants
+            ));
+        return PhpDoc::block($methodDocs);
+    }
+
+    private function magicMethod(): ?PhpClassMember
+    {
+        // Only has streaming RPCs, so exclude __call from implementation, since
+        // the magic method is only for async support at the moment.
+        if ($this->serviceDetails->streamingOnly) {
+            return null;
+        }
+
+        // params
+        $methodVar = AST::var('method');
+        $methodParam = AST::param(null, $methodVar);
+        $argsVar = AST::var('args');
+        $argsParam = AST::param(null, $argsVar);
+        $triggerError = AST::call(AST::TRIGGER_ERROR)(AST::concat('Call to undefined method', AST::__CLASS__, AST::interpolatedString('::$method()')), AST::E_USER_ERROR);
```

Comment:
OH NO! This should be `'Call to undefined method ',`, e.g. it's missing a space after "method"

@noahdietz

Reply:
haha darn! Ok fix incoming